conftest.py

```diff
@@ -68,3 +68,15 @@ def pytest_runtest_setup(item):
 def shuffle_method(request):
     with dask.config.set({"dataframe.shuffle.algorithm": request.param}):
         yield request.param
+
+
+import dask
+from dask.dataframe._compat import PANDAS_GT_150
+
+try:
+    import pyarrow
+except ImportError:
+    pyarrow = False
+
+if PANDAS_GT_150 and pyarrow:
+    dask.config.set({"dataframe.convert_string": True})
```

Comment:
Should be able to remove this now that https://github.com/dask/dask/pull/10017 is in